pdcupdater/tests/handler_tests/test_modules.py

```diff
@@ -2,11 +2,45 @@
 import shutil
 import tarfile
 import copy
+import mock
 
 from pdcupdater.tests.handler_tests import (
     BaseHandlerTest, mock_pdc
 )
 
+def mocked_koji_from_tag(url, tag):
+    if tag != 'epel7':
+        return []
+    return [{
```

Comment:
Hi, I'm a developer of freshmaker now. While I'm reading this patch, I don't quite understand these data. I have two questions.

- name ``mocked_koji_from_tag`` looks like a more generic, and it is used in ``test_handle_ready_state`` now. Are these data is only used for this test, or general for any other tests that require a mocked ``koji_from_tag``?

- which tag attributes are required for running test ``test_handle_ready_state``, or a whole tag dict with all attributes is required?